packages/coinstac-simulator/src/utils/get-computation-registry-stub.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const DecentralizedComputation =
+  require('coinstac-common').models.DecentralizedComputation;
+const path = require('path');
+
+/**
+ * Get computation registry stub.
+ * @module
+ *
+ * Get an object that behaves like `ComputationRegistry` but contains a stubbed
+ * computation via the `computationPath` paramter.
+ *
+ * @param {string} computationPath
+ * @returns {Object}
+ */
+module.exports = function getComputationRegistryStub(computationPath) {
```

Comment:
Provides a `ComputationRegistry`-like stub to avoid the actual class’s registry complexities. Would be worth revisiting and refactoring the `ComputationRegistry` or making simulator use the real deal.